script/setup/install-dev-tools

```diff
@@ -25,7 +25,7 @@ cd "$GOPATH"
 
 # install the `protobuild` binary in $GOPATH/bin; requires module-aware install
 # to pin dependencies
-GO111MODULE=on go get github.com/stevvooe/protobuild
+GO111MODULE=on go get github.com/dims/protobuild@HEAD
```

Comment:
Should eventually move to containerd (not yet, need to get it working first):

```suggestion
GO111MODULE=on go get github.com/containerd/protobuild@HEAD
```

Reply:
will reset this to what was there before